lib/common/models/wp_user/existable.rb

```diff
@@ -76,7 +76,10 @@ def self.display_name_from_body(body)
         # replace UTF chars like  &#187; with dummy character
         title_tag.gsub!(/&#(\d+);/, '|')
 
-        name = title_tag[%r{([^|«»]+) }, 1]
+        name = title_tag[%r{.+» (.+)}, 1]

```

Comment:
In my opinion, actually the best way is to look at the title in some page (e.g. a `404 Not Found` error page) and look for changes.

For example, in my case, the title of a 404 error page is: `Macroarea di Scienze M.F.N. » Pagina non trovata.`. The title asking for author number 1 is `Macroarea di Scienze M.F.N. » admin`.

Saving the substring which is equal and removing it from the author page title we could get the username with no regex or theme dependent issues.